cmd/kube-apiserver/app/server.go

```diff
@@ -553,6 +553,8 @@ func defaultOptions(s *options.ServerRunOptions) error {
 	if err != nil {
 		return fmt.Errorf("error determining service IP ranges: %v", err)
 	}
+	s.SecureServing.ForceLoopbackConfigUsage()
```

Comment:
try adding a congruent line to `federation/cmd/federation-apiserver/app/server.go`.  The federation server can reasonably consider itself the source of truth for namespaces (it mirrors what is happening today) and if they wish to do something different in the future, they'll be able to modify it.